docker/docker-entrypoint.sh

```diff
@@ -34,7 +34,7 @@ echo "$admin_password"
 #seed db
 cd /usr/src/app && psql --host=db --port=5432 --user=fourcat --dbname=fourcat < backend/database.sql
 
-python3 /usr/src/app/helper-scripts/create_user.py -u admin@admin.com -p "$admin_password"
+python3 /usr/src/app/helper-scripts/create_user.py -u admin -p "$admin_password"
```

Comment:
This should also include the `-e` parameter, else it will complain that the username is not a valid e-mail address